### PR TITLE
style: add new file type icons

### PR DESCRIPTION
**Pull request by United600**
Adds new file type icons and more file associations to the list of supported file types, also includes more localization strings in the manifest resources.
Additionally, it updates the hero screenshots in the repository.

- add `ac3` to supported file types
- add `ape` to supported file types
- add `dts` to supported file types
- add `f4v` to supported file types
- add `swf` to supported file types
- add `mxf` to supported file types
- add `nist` to supported file types
- add `spx` to supported file types
- add `ra` to supported file types
- add `vob` to supported file types
- add `wm` to supported file types
- add `wtv` to supported file types
- add unique (type) file type icons to partially address #475
- update the readme with new hero screenshot images

### CODE REVIEW

File: Screenbox/Package.appxmanifest [hunk 1]
```diff
@@ -255,88 +437,110 @@
...
+        <!--<uap:Extension Category="windows.fileTypeAssociation">
+          <uap:FileTypeAssociation Name="wpl">
+            <uap:DisplayName>Windows Media Player Playlist</uap:DisplayName>
+            <uap:EditFlags OpenIsSafe="true"/>
+            <uap:Logo>Assets\Icons\PlaylistFileAssociation.png</uap:Logo>
             <uap:SupportedFileTypes>
-              <uap:FileType ContentType="audio/aiff">.aiff</uap:FileType>
-              <uap:FileType ContentType="audio/aiff">.aif</uap:FileType>
-              <uap:FileType ContentType="audio/aiff">.aifc</uap:FileType>
+              <uap:FileType ContentType="application/vnd.ms-wpl">.wpl</uap:FileType>
```
Comment (huynhsontung) [inline Screenbox/Package.appxmanifest:512]: Same here. Why did this file type association get commented out? What's the plan for these file types?

File: Screenbox/Package.appxmanifest [hunk 2]
```diff
@@ -255,88 +437,110 @@
...
-          <uap:FileTypeAssociation Name="m3u8">
+        <!--<uap:Extension Category="windows.fileTypeAssociation">
+          <uap:FileTypeAssociation Name="dvr-ms">
+            <uap:DisplayName>Microsoft Digital Video Recording</uap:DisplayName>
+            <uap:EditFlags OpenIsSafe="true"/>
+            <uap:Logo>Assets\Icons\VideoFileAssociation.png</uap:Logo>
             <uap:SupportedFileTypes>
-              <uap:FileType ContentType="application/vnd.apple.mpegurl">.m3u8</uap:FileType>
-              <uap:FileType ContentType="audio/mpegurl">.m3u</uap:FileType>
+              <uap:FileType>.dvr-ms</uap:FileType>
```
Comment (huynhsontung) [inline Screenbox/Package.appxmanifest:472]: Why is this file type association commented out?

File: README.md [hunk 3]
```diff
@@ -21,9 +21,13 @@ Screenbox is a modern video player that cares about performance and ease of use
...
 
-![Screenshot of the home page](https://user-images.githubusercontent.com/31434093/226089502-0b82157d-8e48-408c-b501-6b6c17b8a584.png)
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="../../../Screenbox/blob/assets/assets/images/docs/PC-HeroHomePage-Dark.png?raw=true">
+  <source media="(prefers-color-scheme: light)" srcset="../../../Screenbox/blob/assets/assets/images/docs/PC-HeroHomePage-Light.png?raw=true">
+  <img alt="Screenshot of the Screenbox home page." src="../../../Screenbox/blob/assets/assets/images/docs/PC-HeroHomePage-Dark.png?raw=true" width="1248">
+</picture>
 
-![Screenshot of the video player](https://user-images.githubusercontent.com/31434093/226089522-fc02208d-a7b5-4821-bb74-f48f79e9c813.png)
+![Screenshot of the Screenbox video player](/assets/images/docs/PC-HeroPlayerPageVideo_Dark.png)
```
Comment (huynhsontung) [inline README.md:30]: Very nice touch! 👍👍

File: Screenbox/Package.appxmanifest (hunk 1 above)
Comment (United600) [reply Screenbox/Package.appxmanifest:512]: Wasn't able to test it and I added it in so I don't forget the extension

File: README.md (hunk 3 above)
Comment (United600) [reply README.md:30]: The relative links may require some adjustments

Comment (huynhsontung) [reply README.md:30]: Can't we just have full absolute links instead of relative links?

Comment (huynhsontung) [reply README.md:30]: Oh nevermind I just realized that doesn't adapt to the current branch.

**Review by huynhsontung — APPROVED** [review]

**Pull request merged**